ci/gpu/build.sh

```diff
@@ -73,6 +75,9 @@ pip install "git+https://github.com/dask/distributed.git@main" --upgrade --no-de
 pip install "git+https://github.com/dask/dask.git@main" --upgrade --no-deps
 set +x
 
+# Install pre-built conda packages from previous CI step
+gpuci_logger "Install libraft conda packages from CPU job"
+gpuci_mamba_retry install --use-local "${LIBRAFT_CONDA_PACKAGES}/libraft*.bz2"
```

Comment:
This line could potentially cause issues. From the `conda install` [docs here](https://docs.conda.io/projects/conda/en/latest/commands/install.html):

> conda can also be called with a list of explicit conda package filenames (e.g. ./lxml-3.2.0-py27_0.tar.bz2). Using conda in this mode implies the --no-deps option, and should likewise be used with great caution.

So the implication of calling `conda install` on an explicit filename is that it will only install the package itself and not any of the package's runtime dependencies.

The solution to this problem is to install the package using the `--channel`/`-c` flag with a value that points to the local `conda` channel. So something like this:

```sh
export LIBRAFT_CONDA_PACKAGES="$WORKSPACE/ci/artifacts/raft/cpu/.conda-bld/" # notice there is no `linux-64` here
gpuci_mamba_retry install -c "${LIBRAFT_CONDA_PACKAGES}" libraft
```

This will tell `conda install` to do install `libraft` from the channel on the local filesystem and to also install `libraft`'s dependencies as it usually would.